radix-transactions/src/manifest/static_resource_movements/typed_invocation.rs

```diff
@@ -4,18 +4,17 @@ use radix_engine_interface::blueprints::consensus_manager;
 use radix_engine_interface::blueprints::identity;
 use radix_engine_interface::blueprints::locker;
 
+use super::*;
 use radix_common::prelude::*;
-use radix_engine_interface::prelude::*;
 
 macro_rules! define_typed_invocations {
     (
         $(
             $package_name: ident => {
+                package: $package_address: expr,
                 $(
                     $blueprint_name: ident => {
-                        type: $type: ty,
                         entity_type_pat: $entity_type_pat: pat,
-                        module_id: $module_id: expr,
```

Comment:
Because this isn't the right abstraction here - the `module_id` for these methods is always `main`. Instead, a module id of e.g. Royalty, should redirect at resolution to the blueprint of the royalty module 👍